### PR TITLE
Fix processing tests

### PR DESCRIPTION
**Pull request by SylviaWhittle**
This PR (hopefully) fixes the tests in `test_processing.py` for the #932

### CODE REVIEW

**Comment by SylviaWhittle** [comment]
Ah heck, so this PR is just to address the processing tests, not the others... Any ideas on how to do a PR that fixes only some of the tests? @ns-rse

**Comment by SylviaWhittle** [comment]
The processing tests pass on my machine, you might want to verify though 
image

**Comment by ns-rse** [comment]
> Any ideas on how to do a PR that fixes only some of the tests? ￼@ns-rse

The Continuous Integration tests will continue to fail until all tests pass, no way around that I can think of without switching to using `pytest --testmon` in CI if that is even possible as it would be problematic if say a newer package were installed in CI than on a local machine where the `.testmondata`.

Its useful to run the whole test suite in CI though and ensuring it runs on _all_ pull requests was something I deliberately introduced the other day with #921 as had this been the case from the start (rather than just running on PRs that target `main`) we would have known about and addressed the failing tests earlier.

**Review by MaxGamill-Sheffield — APPROVED** [review]
Looks good and TS runs!

File: topostats/grains.py [hunk 1]
```diff
@@ -295,6 +300,41 @@ def remove_small_objects(self, image: np.array, **kwargs) -> npt.NDArray:
             return small_objects_removed > 0.0
         return image
 
+    def remove_objects_too_small_to_process(
+        self, image: npt.NDArray, minimum_size_px: int, minimum_bbox_size_px: int
```
Comment (ns-rse) [inline topostats/grains.py:304]: Do we expect this method to be used independently of the processing?

If not then I don't see any advantage have `minimum_size_px` or `minimum_bbox_size_px` as arguments to the method as they are set as attributes to the class when it is initialised and available as `self.minimum_grain_size_px` and `self.minimum_bbox_size_px`.

The comment above on hardcoding suggests its not something that will ever be changed.

**Pull request merged**